quinn-proto/src/connection/datagrams.rs

```diff
@@ -20,26 +20,38 @@ impl<'a> Datagrams<'a> {
     /// Queue an unreliable, unordered datagram for immediate transmission
     ///
     /// Returns `Err` iff a `len`-byte datagram cannot currently be sent
-    pub fn send(&mut self, data: Bytes) -> Result<(), SendDatagramError> {
+    ///
+    /// If `drop` is true, previously queued datagrams which are still unsent may be discarded to
```

Comment:
Let's move the "Returns `Err`" clause below the "If `drop` is true" clause and make sure we mention that this will only be emitted if `drop` is `false`? (Also please end it in a period.)